src/components/ExportButton/exportUtils.js

```diff
@@ -324,9 +329,14 @@ export const exportPdf = async (
   // Add canvas to PDF
   doc.addImage(canvas, 'JPEG', 0, 0, exportSize[0], exportSize[1]);
 
+  const topicFileName = topic?.exportConfig?.exportFileName;
+
   // download the result
   const filename = `${
-    topic?.exportConfig?.exportFileName ||
```

Comment:
The exportConfig object makes me nervous in this function :D   can you add exportFileName and overlyImageUrl as parameters) to this function . So we remove the topic  and the i18n object as parameter. then the translation happens at the component level where we already have i18n and t

Reply:
Done